charts/image-resizer/templates/deployment.yaml

```diff
@@ -56,6 +56,13 @@ spec:
               scheme: HTTP
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          envFrom:
+          - configMapRef:
+              name: {{ include "image-resizer.fullname" . }}
```

Comment:
does this configmap exist? it doesn't look like it

Reply:
Oh yes, I've added it now